packages/wonder-blocks-dropdown/src/components/__docs__/single-select.stories.js

```diff
@@ -150,6 +150,7 @@ const items = [
     <OptionItem label="Grape" value="grape" key={6} />,
     <OptionItem label="Lemon" value="lemon" key={7} />,
     <OptionItem label="Mango" value="mango" key={8} />,
+    <OptionItem label="" value="" key={9} />,
```

Comment:
**suggestion:** This change should be reverted since it affects existing stories.  For the story you've added, you could do `{[...items, <OptionItem label="" value="" key={9} />]}`.

Reply:
Updated!